packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -90,35 +99,168 @@ class BottomNavigationBar extends StatefulWidget {
   /// The index into [items] of the current active item.
   final int currentIndex;
 
-  /// Defines the layout and behavior of a [BottomNavigationBar].
+  /// Defines the layout and behavior of a [BottomNavigationBar].  See
```

Comment:
put a line break and blank line between the first sentence (used as a summary in the docs) and subsequent sentences that point to other things or give more information.

also, generally we use just one space after periods. HTML strips out any additional spaces anyway.

Reply:
Done.  Sorry, I do know about the one-space thing, I just missed that one.  My fingers type two automatically.

Is there any magic to the words "See also:" in the docs?  Should the second sentence here do that?

Reply:
No particular magic, just convention. Use whichever works best.